### DISCUSSION TITLE
Welcome to bulk-downloader-for-reddit Discussions!

### DISCUSSION THREAD

**Discussion by aliparlakci** [Serene-Arc/bulk-downloader-for-reddit#246] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MuhammadEhsan2** [comment]


**Reply by vladdoster** [reply]
@MuhammadEhsan2 

I think the [configuration section](https://github.com/aliparlakci/bulk-downloader-for-reddit#configuration) in the README.md has what you're looking for.
